### PR TITLE
[#195] Regexp optimization pattern

### PR DESCRIPTION
**Pull request by Doldrums**

### CODE REVIEW

File: lib/pdd/source.rb [hunk 1]
```diff
@@ -37,10 +37,13 @@ def initialize(file, path)
       @path = path
     end
 
-    def match_markers(line)
-      MARKERS.map do |mkr|
-        %r{(.*(?:^|\s))#{mkr}\s+#([\w\-.:/]+)\s+(.+)}.match(line)
-      end.compact
+    def match_markers(l)
```
Comment (yegor256) [inline lib/pdd/source.rb:40]: @Doldrums what's wrong with `line`? Why renaming to `l`?

**Comment by yegor256** [comment]
@Doldrums see my comment above

File: lib/pdd/source.rb (hunk 1 above)
Comment (Doldrums) [reply lib/pdd/source.rb:40]: we have too strict rules for the length of the string, and because of the length of the regular, had to correct

**Comment by yegor256** [comment]
@rultor merge

**Comment by Doldrums** [comment]
@yegor256 fixed

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**